elements/pfe-button/README.md

```diff
@@ -0,0 +1,103 @@
+# PatternFly Element | Button Element
+This component is a web component wrapper around a standard HTML Button Element.
+`<pfe-button>` is heavily influenced by the findings in this post: [Custom
+elements, shadow DOM and implicit form submission](https://www.hjorthhansen.dev/shadow-dom-and-forms/). You can expect `<pfe-button>` to work like a normal
+HTML Button Element.
+
+## Usage
+When using this component, you must provide a standard HTML Button Elements as
+the only light DOM child of `pfe-button`.
+
+```html
+<pfe-button>
+  <button>My Button</button>
+</pfe-button>
+```
+
+### Accessibility
+`<pfe-button>` addresses the issues associated with typical implementations of
+web component buttons. When using a `<pfe-button>` in a `<form>` element, the
+`<pfe-button>` will function as a standard button in a `<form>`. You can expect
+the button to submit the form.
+
+#### Disabled Attribute
+Adding the `disabled` attribute to either the `<pfe-button>` wrapper or the
+`<button>` element in the light DOM will disable the button.
+
+```html
+<pfe-button disabled>
+  <button>Submit</button>
+</pfe-button>
+```
+
+Or
+
+```html
+<pfe-button>
+  <button disabled>Submit</button>
+</pfe-button>
+```
+
+#### Type Attribute
+Using the `type` attribute works in the same fashion as the `disabled`
+attribute. You can add a `type` attribute to either the `<pfe-button>` wrapper
+or the `<button>` element in the light DOM.
+
+```html
+<pfe-button type="button or reset or submit">
+  <button>Submit</button>
+</pfe-button>
+```
+
+Or
+
+```html
+<pfe-button>
+  <button type="button or reset or submit">Submit</button>
+</pfe-button>
+```
+
+## Slots
+
+None
+
+## Attributes
+`disabled` (observed): Disables the button
+
+`pfe-variant`: Changes the style of the button. Possible values are
+- primary (default)
+- secondary
+- tertiary
+- danger
+- control
+
+## Events
+### pfe-button:click
```

Comment:
```suggestion
### pfe-button:click
```

Click is a mouse-specific phrase.  Can we use something more generic like `fire` since it's triggered by key press or mouse?